### ISSUE TITLE
Chart Responsiveness

### ISSUE THREAD

**Issue by rortega92** [benkeen/d3pie#60]
Hello, first and most importantly, thank you for such an amazing tool its been really useful. Do you have any plans on adding chart responsiveness? Or could you tell me how can I achieve this? The chart right now is based on a fixed size that doesn't rely on the container it is being binded to. Many sites are based on responsive design and a tool like yours could be really useful if it could be responsive. Thank you!

**Comment by BorgPhysics** [comment]
Agreed.  It would be nice to have an option to automatically fit within the parent element's width and height.

**Comment by willthemoor** [comment]
```css
.containing-element svg { 
    max-width: 100%;
}
```

works for me. Thing is, depending on how many slices there, it's not exactly "usable" on a small screen. But that should at least contain it.

**Comment by jeremydiba** [comment]
I would think this to be a priority feature.  I tried using the workaround above posted by @willthemoor to no avail.  If someone could go more in depth on a workaround to this issue I would appreciate it.
